src/_pytest/pytester.py

```diff
@@ -130,9 +125,9 @@ def isopen(line):
 
         return open_files
 
-    def matching_platform(self):
+    def matching_platform(self) -> bool:
         try:
-            subprocess.check_output(("lsof", "-v"))
```

Comment:
just out of curiosity, why do you prefer `run` to `check_output` here?

Reply:
There's no good reason, only that the [subprocess docs](https://docs.python.org/3/library/subprocess.html) start with describing only the `run` function and say

> The run() function was added in Python 3.5; if you need to retain compatibility with older versions, see the Older high-level API section.

And that "Older high-level API" includes `check_output`. So I just use `run` for everything.

Reply:
thanks for explaining @bluetech

Reply:
😁